### PR TITLE
Rename to "T2-Project", rewrite TimeoutCollector, fix grammar

### PR DESCRIPTION
**Pull request by delvh**

### CODE REVIEW

File: src/test/java/de/unistuttgart/t2/cart/repository/CollectorTests.java [hunk 1]
```diff
@@ -17,7 +17,8 @@
 @SpringBootTest
 @ActiveProfiles("test")
 /**
- * actually i was trying to test my collector.. but timing is a bad bitch and it fucks me up.
+ * Actually, I was trying to test my collector.<br>
```
Comment (nk-coding) [inline src/test/java/de/unistuttgart/t2/cart/repository/CollectorTests.java]: Maybe not only reformat this but remove this

File: src/main/java/de/unistuttgart/t2/cart/repository/TimeoutCollector.java [hunk 2]
```diff
@@ -20,12 +21,17 @@
...
-public class TimeoutCollector {
+public final class TimeoutCollector {
+
+    private static final Logger LOG = LoggerFactory.getLogger(TimeoutCollector.class);
 
     private final long TTL; // seconds
     private final int taskRate; // milliseconds
-
-    private final Logger LOG = LoggerFactory.getLogger(getClass());
+    final Runnable cleanup = () -> {
```
Comment (nk-coding) [inline src/main/java/de/unistuttgart/t2/cart/repository/TimeoutCollector.java]: maybe a method would be a cleaner solution

**Review by nk-coding — COMMENTED** [review]
.

File: src/main/java/de/unistuttgart/t2/cart/repository/TimeoutCollector.java (hunk 2 above)
Comment (delvh) [reply src/main/java/de/unistuttgart/t2/cart/repository/TimeoutCollector.java]: Yep, I also noticed that one later in `inventory`(?) where the exact same thing needs to be present.

**Comment by delvh** [comment]
@nk-coding ping (for you as well as for me).

**Review by nk-coding — APPROVED** [review]

**Comment by delvh** [comment]
:rofl:

**Pull request merged**